app/poros/about_info.rb

```diff
@@ -0,0 +1,13 @@
+class AboutInfo
+  attr_reader :name,
+              :bio,
+              :email,
+              :image
+
+  def initialize(data)
+    @name = data[:name]
+    @bio  = data[:bio].sub('~', ',')
```

Comment:
So the bio can have commas in it, in the CSV they'll have to be `~`'s so they can be turned into `,`'s here.